test/src/test/java/jenkins/model/JenkinsManagePermissionTest.java

```diff
@@ -207,15 +219,39 @@ public void globalConfigAllowedWithManagePermission() throws Exception {
 
         HtmlForm form = j.createWebClient().goTo("configure").getFormByName("config");
         HtmlPage updated = j.submit(form);
-        assertEquals("User with Jenkins.MANAGE permission should be able to update global configuration",
-                     200, updated.getWebResponse().getStatusCode());
+        assertThat("User with Jenkins.MANAGE permission should be able to update global configuration",
+                     updated.getWebResponse(), hasResponseCode(HttpURLConnection.HTTP_OK));
     }
 
     private String getShell() {
         Descriptor descriptorByName = j.getInstance().getDescriptorByName("hudson.tasks.Shell");
         return ((Shell.DescriptorImpl) descriptorByName).getShell();
     }
 
+    private Matcher<WebResponse> hasResponseCode(final int httpStatus) {
+        return new BaseMatcher<WebResponse>() {
+            @Override
+            public boolean matches(final Object item) {
+                final WebResponse response = (WebResponse) item;
+                return (response.getStatusCode() == httpStatus);
+            }
+
+            @Override
+            public void describeTo(final Description description) {
+                description.appendText("Jenkins to return  ").appendValue(httpStatus);
+            }
+
+            @Override
+            public void describeMismatch(Object item, Description description) {
+                WebResponse response = (WebResponse) item;
+                description.appendText("Response code was: ");
+                description.appendValue(response.getStatusCode());
+                description.appendText(" with error message: ");
+                description.appendText(response.getStatusMessage());
```

Comment:
would the page content be useful?

Reply:
If not, it would provide the same information than the standard `equalTo` matcher (or whatever its name is), the value you expect vs the current value.

What do you think we can provide to make this description more clear?

Reply:
^ Forget that, I misunderstood your comment :smile: